### PR TITLE
Add carousel with quotes

### PR DESCRIPTION
**Pull request by banciur**
At shit moment it is disabled as we don't have content for it.

carousel_desktop
carousel_mobile

### CODE REVIEW

File: common/sass/variables.scss [hunk 1]
```diff
@@ -18,6 +18,7 @@ $black: #000;
 $black-lighter: #262B2F;
 $green: #D5E20F;
 $white-green: #F6FAF3;
+$white-almost: #f7f8f9;
```
Comment (krzkaczor) [inline common/sass/variables.scss]: this color is already used in common.scss (`$lightgray: #f7f8f9;`) Could you remove duplication? probably common.scss should use your new defined color. Also please, make it uppercase as others are also uppercased (i am taking care of `off-white` on my pr)

File: page/sass/quotes.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+@import '../../common/sass/variables';
```
Comment (krzkaczor) [inline page/sass/quotes.scss]: this is not needed. (its needed only with css modules in react)

File: page/partials/quotes.ejs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<div class="section quotes">
+  <div class="row">
+    <div class="col-md-6">
+      <h3><%= title %></h3>
```
Comment (krzkaczor) [inline page/partials/quotes.ejs]: is h3 without any additional styles look good or should i style it since i do it for all other elements?

Comment (banciur) [reply page/partials/quotes.ejs]: Yes it looks same as other section titles

File: common/sass/variables.scss (hunk 1 above)
Comment (banciur) [reply common/sass/variables.scss]: I didn't make it uppercase as according to sass linter it should be lowercase. In separate PR I will fix whole file to follow litner rules

**Pull request merged**